redux/actions/adminRequests.js

```diff
@@ -0,0 +1,24 @@
+import http from "../../src/utils/axios";
+import { GET_REQUESTS, GET_REQUESTS_ERROR} from "./types";
+
+export const adminRequests = () => async(dispatch) => {
+  try {
+    let token= localStorage.getItem('token')
+    const res = await http.get(`/api/request/manager`,{
+      headers:{
+        "x-auth-token": token
+      }
+    });
```

Comment:
Remove the `x-auth-token `header as it's already included in` axios configuration`. 
Also delete `token` variable,we are using `x-auth-token` header configured in axios .